stratum/chassis_config.pb_with_PortShaping.txt.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: Copyright 2020-present Open Networking Foundation.
+SPDX-License-Identifier: Apache-2.0
```

Comment:
Add as comment directly in file (`# comment`)

Reply:
thanks. I already resolved.